package-lock.json

```diff
@@ -1,4724 +1,8 @@
 {
   "name": "@workos-inc/node",
-  "version": "2.8.1",
-  "lockfileVersion": 2,
+  "version": "2.9.1",
+  "lockfileVersion": 1,
```

Comment:
Any idea why this went from version `2` to `1`?

Reply:
I think someone might've ran `npm i` using NodeV16.

Reply:
Someone is renovate-bot: https://github.com/workos/workos-node/pull/641/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519R4

Reply:
We should lock it down so that renovate uses the same version as our other projects.